package.json

```diff
@@ -22,7 +22,8 @@
     "wo-imap-handler": "~0.1.11",
     "mimefuncs": "~0.3.4",
     "axe-logger": "~0.0.2",
-    "tcp-socket": "~0.4.0"
+    "tcp-socket": "~0.4.0",
+    "es6-promise": "^2.0.1"
```

Comment:
i don't know if this is a good idea. including polyfills in your dependencies is generally considered a bad practice and i don't think we should do it. polyfills belong into the application-developer's hands and should not be part of the library... what do you think?